app/src/test/java/com/nervousfish/nervousfish/modules/pairing/APairingHandlerTest.java

```diff
@@ -69,25 +80,56 @@ void send(byte[] buffer) {
         }
     }
 
+    private byte[] serializeFile() throws IOException {
+        final RandomAccessFile f = new RandomAccessFile(constants.getDatabaseContactsPath(), "rw");
+        final byte[] fileData = new byte[(int) f.length()];
+        final byte[] bytes;
+        f.readFully(fileData);
+        final FileWrapper fWrapper = new FileWrapper(fileData);
+        try (ByteArrayOutputStream bos = new ByteArrayOutputStream();
+             ObjectOutputStream oos = new ObjectOutputStream(bos)) {
+            oos.writeObject(fWrapper);
+            oos.flush();
+            bytes = bos.toByteArray();
+        }
+        return bytes;
+    }
+
     private PairingHandler tmp, phspy;
     private IDatabase database;
+    private IConstants constants;
+    private IServiceLocator serviceLocator;
 
     @Before
     public void setup() throws Exception {
-        IServiceLocator serviceLocator = mock(IServiceLocator.class);
+        serviceLocator = mock(IServiceLocator.class);
         database = mock(IDatabase.class);
         when(serviceLocator.getDatabase()).thenReturn(database);
+        constants = mock(IConstants.class);
+        when(serviceLocator.getConstants()).thenReturn(constants);
+        when(constants.getDatabaseContactsPath()).thenReturn(new File("E:\\docs\\NervousFish\\app\\src\\main\\AndroidManifest.xml").getAbsolutePath());
```

Comment:
This path is very specific and won't work on all machines. Can't you use a relative path?